collect_app/src/main/res/values/typography.xml

```diff
@@ -44,7 +44,7 @@
         <item name="android:textColor">?colorPrimary</item>
     </style>
 
-    <style name="TextAppearanceLabelExtraLarge" parent="TextAppearance.Material3.LabelLarge">
+    <style name="TextAppearance.Collect.LabelExtraLarge" parent="TextAppearance.Material3.TitleLarge">
```

Comment:
If now it extends TitleLarge wouldn't it be better to update the title too?
`textAppearanceLabelExtraLarge` -> `textAppearanceTitleLarge`
`TextAppearance.Collect.LabelExtraLarge` - > `TextAppearance.Collect.TitleLarge`

Reply:
We're not actually changing `?textAppearanceTitleLarge` though - just using it as the base for our custom "Label Extra Large" style.